VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:
VAL_TABLE_ Board_ID_2 7 "Board 6" 5 "Board 4" 3 "Board 8" 1 "Board 2" ;
VAL_TABLE_ Board_ID_1 6 "Board 7" 4 "Board 5" 2 "Board 3" 0 "Board 1" ;


BO_ 1800 PDM_CAN_TX_8: 6 Vector__XXX
 SG_ Board8_Temp : 32|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current2 : 16|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current1 : 0|16@1- (1,0) [0|0] "" Vector__XXX

BO_ 1801 PDM_CAN2_TX_8: 6 Vector__XXX
 SG_ Board8_Temp :  33|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current2 : 17|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current1 : 1|16@1- (1,0) [0|0] "" Vector__XXX

BO_ 1802 PDM_CAN3_TX_8: 6 Vector__XXX
 SG_ Board8_Temp :  32|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current2 : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current1 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

 BO_ 1803 PDM_CAN4_TX_8: 6 Vector__XXX
 SG_ Board8_Temp : 32|16@1- (0.1,0) [0|0] "" Vector__XXX
 SG_ Board8_Current2 : 16|16@1- (4,0) [0|0] "" Vector__XXX
 SG_ Board8_Current1 : 0|16@1- (1,10) [0|0] "" Vector__XXX


BA_DEF_  "MultiplexExtEnabled" ENUM  "No","Yes";
BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "MultiplexExtEnabled" "No";
BA_DEF_DEF_  "BusType" "CAN";

